src/convertToObject.js

```diff
@@ -3,16 +3,30 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in
+ * [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrayOfStrings = sourceString.split(';');
+  const object = {};
+
+  for (const el of arrayOfStrings) {
+    const currentKey = el.slice(0, el.indexOf(':')).trim().replace(/_/g, '-');
+    const currentValue = el.slice(el.indexOf(':') + 1).trim();
```

Comment:
All chained methods called on the same level should have same indentation (Checklist #4)